src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -697,18 +703,20 @@ private Acl getAcl(Connection conn, String uniqueId) throws SQLException {
     try (PreparedStatement stmt = getAclFromDb(conn, uniqueId);
```

Comment:
Perhaps rename getAclFromDb to runAclSqlStatement (to disambiguate that it is not running content sql)?

Reply:
As I mentioned, I didn't entirely like the structure of getAclFromDb, but it has parallel methods for getDocFromDb and getStreamFromDb. All three methods just create a PreparedStatement, so a basic rename to prepare{Acl,Doc,Stream}Statement seems reasonable, but feels like a separate CL. Maybe we can finish this one and start a new pull request to massage the structure?

Also, these methods have functional utility for try-with-resources: we want to call prepareStatement and executeQuery but do something in between. So we either need these helper methods, or we need two nested try-with-resources.

Reply:
Ah, yes, that changed here https://github.com/googlegsa/database/commit/afe59e4949b506bf81e55c230448749dd9e54ab9#diff-dd03438283161f00e652d4769de767bf

I agree that some structure/renaming changes are fitting.  

I have made a client and am reading DatabaseAdaptor.java file to  re-acquaint myself with it. I plan to read it on Monday and make some suggestions then.

Thank you